python/tank/platform/engine.py

```diff
@@ -2226,12 +2214,9 @@ def _initialize_dark_look_and_feel(self):
         if self.has_qt5:
             self.log_debug("Applying Qt5-specific styling...")
             self.__initialize_dark_look_and_feel_qt5()
-        elif self.has_qt4:
```

Comment:
Should we need to add another condition for qt6? I don't have context here but just writing this to double check.